CSquared/LexicalAnalyzer/Lexer.cs

```diff
@@ -444,16 +444,6 @@ public ILexeme Lex()
                                 LinePosition = this.Reader.LinePosition
                             };
                         }
-                        else if (currentLetters == "var")
-                        {
-                            return new Lexeme
-                            {
-                                Type = LexemeType.Var,
-                                Value = currentLetters,
-                                LineNumber = this.Reader.LineNumber,
-                                LinePosition = this.Reader.LinePosition
-                            };
-                        }
                         else if (currentLetters == "async")
```

Comment:
this needs to remain. it looks like you removed it... (removing this would force the use of python variables, which reduces the language's expressiveness. leaving it with the above change gives the option to use var, or not)